pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -370,27 +370,18 @@ func (p *PriorityQueue) flushBackoffQCompleted() {
 			return
 		}
 		pod := rawPodInfo.(*framework.PodInfo).Pod
-		boTime, found := p.podBackoff.GetBackoffTime(nsNameForPod(pod))
-		if !found {
-			klog.Errorf("Unable to find backoff value for pod %v in backoff queue", nsNameForPod(pod))
-			p.podBackoffQ.Pop()
+		if !p.isPodBackingOff(pod) {
```

Comment:
This is semantically different from the current code. The current code continues to move the pod even if it was not found in the podBackoff map. 

I don't know though if there are corner cases where the podBackoff queue and map are out of sync, in which case we should stick with the current logic.

Reply:
Thanks for review, I would take a closer look at the current logic to check if inconsistency may exist.